prow/scripts/cluster-integration/helpers/eventing.sh

```diff
@@ -172,3 +172,32 @@ function eventing::test_fast_integration_eventing() {
 
     log::success "Eventing tests completed"
 }
+
+function eventing::pre_upgrade_test_fast_integration_eventing() {
+    log::info "Running pre upgrade Eventing E2E release tests"
+
+    pushd /home/prow/go/src/github.com/kyma-project/kyma/tests/fast-integration
+    make ci-test-eventing-pre-upgrade
+    popd
+
+    log::success "Pre upgrade Eventing tests completed"
+}
+function eventing::fast_integration_tests() {
+    log::info "Running only Eventing E2E release tests"
+
+    pushd /home/prow/go/src/github.com/kyma-project/kyma/tests/fast-integration
+    make ci-test-eventing-post-tests
```

Comment:
Somehow I don't see this target here https://github.com/kyma-project/kyma/pull/12905! Where is this make target? on `main`?

Reply:
fixed